<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegeditManageTool</class>
 <widget class="QMainWindow" name="RegeditManageTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册表管理工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/winIcon</normaloff>:/images/winIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gLyAddRBTool" columnstretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="lblIconPath">
       <property name="text">
        <string>图标路径</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lEdtExecPath"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pBtnGetExecPath">
       <property name="text">
        <string>打开文件目录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lEdtTooName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblKeyName">
       <property name="text">
        <string>工具名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lEdtIconPath"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblExecPath">
       <property name="text">
        <string>执行路径</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pBtnGetIconPath">
       <property name="text">
        <string>打开文件目录</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pBtnAddTool">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pBtnDelTool">
       <property name="text">
        <string>移除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>26</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
